setup.cfg

```diff
@@ -50,7 +50,7 @@ include =
 max-line-length = 120
 
 [mypy]
-python_version = 3.7
+python_version = 3.10
```

Comment:
Do you know what was causing this to fail with 3.7? With it set to a newer version, it will pass for standard library functions that only exist in the newer versions but not in older versions, so can break backwards compatibility. 3.7 is EOL in June, so definitely don't need to keep support past that, but we can also bump the minimum version if there is a good reason (3.7 is definitely getting pretty old).

Reply:
Good point - I didn't consider this downside. I was getting an error which I see now you've fixed (https://github.com/flacjacket/pywayland/commit/88de992dba7997f77000db93d269f72580ab01d5). Will revert.